cluster/gce/config-default.sh

```diff
@@ -521,3 +521,6 @@ export GCE_PRIVATE_CLUSTER_PORTS_PER_VM="${KUBE_GCE_PRIVATE_CLUSTER_PORTS_PER_VM
 # Optional: Create apiserver konnectivity server and agent.
 export ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE="${KUBE_ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE:-false}"
 export KONNECTIVITY_SERVICE_PROXY_PROTOCOL_MODE="${KUBE_KONNECTIVITY_SERVICE_PROXY_PROTOCOL_MODE:-grpc}"
+
+# Optional: Enable Windows CSI-Proxy
+ENABLE_CSI_PROXY="${ENABLE_CSI_PROXY:-true}"
```

Comment:
why is there no "export"?

Reply:
I see there are some variable using export, some don't e.g., WINDOWS_NODE_TAINTS="${WINDOWS_NODE_TAINTS:-node.kubernetes.io/os=win1809:NoSchedule}"
 I am actually not sure in which condition it should use it.

Reply:
I think in our script setup, since there is no new process spawned, export seems not necessary/

Reply:
i guess one way to tell is to bring up a cluster, see if the env you added is passed properly. that said, i think it does not hurt to add 'export'.

Reply:
I tried it out, and it is working fine. But agree with you, no harm to add export. So added it.PTAL